x-pack/platform/plugins/shared/lens/public/react_embeddable/data_loader.test.ts

```diff
@@ -117,12 +117,6 @@ async function expectRerenderOnDataLoader(
     onBrushEnd: jest.fn(),
     onFilter: jest.fn(),
     onTableRowClick: jest.fn(),
-    // Make TS happy
```

Comment:
These are no longer needed because parentApi no longer has to implement `PresentationContainer`.